src/main/java/org/opensearch/security/auditlog/sink/WebhookSink.java

```diff
@@ -352,33 +356,36 @@ public boolean isTrusted(X509Certificate[] chain, String authType) {
 
 	    try {
 
+			HttpClientBuilder hcb = HttpClients.custom().setDefaultRequestConfig(config);
 	        if(!verifySSL) {
-	            return HttpClients.custom()
-	                    .setSSLSocketFactory(
-	                            new SSLConnectionSocketFactory(
-	                                    new SSLContextBuilder()
-	                                    .loadTrustMaterial(trustAllStrategy)
-	                                    .build(),
-	                                    NoopHostnameVerifier.INSTANCE))
-	                    .setDefaultRequestConfig(config)
-	                    .build();
+				SSLContext sslContext = SSLContextBuilder.create().loadTrustMaterial(trustAllStrategy).build();
+				final SSLConnectionSocketFactory sslsf = new SSLConnectionSocketFactory(sslContext, null, null,
+						NoopHostnameVerifier.INSTANCE);
+
+				final HttpClientConnectionManager cm = PoolingHttpClientConnectionManagerBuilder.create()
+						.setSSLSocketFactory(sslsf)
+						.setDefaultSocketConfig(SocketConfig.custom().setSoTimeout(60, TimeUnit.SECONDS).build())
```

Comment:
`SocketConfig.custom().setSoTimeout(timeout, TimeUnit.SECONDS)`  as per https://github.com/opensearch-project/security/pull/2166/files#diff-915d8d93aacc6f1f20d9e49a89cd35a69599cbb07a8ff707d823dd3cb528ad16L344 ?

Reply:
Changed to use `timeout` variable.